--- libsbsms-2.0.2/configure.ac.orig	2011-07-02 08:01:36.000000000 +0200
+++ libsbsms-2.0.2/configure.ac	2011-07-10 14:54:00.955720515 +0200
@@ -82,9 +82,9 @@
 fi
 
 if test x$enable_debug = xyes; then
-   SBSMS_CFLAGS="$SBSMS_CFLAGS -g -O3"
+   SBSMS_CFLAGS="$SBSMS_CFLAGS -g"
 else
-   SBSMS_CFLAGS="$SBSMS_CFLAGS -O3 -fomit-frame-pointer"
+   SBSMS_CFLAGS="$SBSMS_CFLAGS -fomit-frame-pointer"
 fi
 
 if test x$enable_multithreaded = xyes; then
